features/importers/venepaikka_harbors/importer.py

```diff
@@ -0,0 +1,210 @@
+from typing import Iterable
+
+import jmespath
+import requests
+from django.conf import settings
+from django.contrib.gis.geos import Point
+from django.utils import timezone
+
+from categories.models import Category
+from features.enums import FeatureTagSource
+from features.importers.base import FeatureImporterBase
+from features.importers.venepaikka_harbors import app_settings
+from features.models import ContactInfo, Feature, Image, License, SourceType, Tag
+
+query = """
+query Harbors {
+  harbors {
+    edges {
+      node {
+        id
+        geometry {
+          type
+          coordinates
+        }
+        properties {
+          name
+          imageLink
+          imageFile
+          streetAddress
+          zipCode
+          municipality
+          phone
+          email
+          servicemapId
+        }
+      }
+    }
+  }
+}
+"""
+
+feature_expression = jmespath.compile(
+    """
+data.harbors.edges[*].node.{
+    id: id,
+    name: properties.name,
+    lat: geometry.coordinates[1],
+    lon: geometry.coordinates[0],
+    address: {
+        street_address: properties.streetAddress,
+        postal_code: properties.zipCode,
+        municipality: properties.municipality
+        phone_number: properties.phone
+        email: properties.email
+    },
+    images: [properties.imageFile, properties.imageLink]
+    servicemap_id: properties.servicemapId
+}
+"""
+)
+
+
+class VenepaikkaImporter(FeatureImporterBase):
+
```

Comment:
so, do you prefer to have empty lines before attributes' declarations in a class or not? differs between classes within this commit 👀

Reply:
Same issue also on the `MyHelsinkiImporter`. But no worries, a fix is on the way. :wrench: